themes/digital.gov/layouts/partials/core/get_related.html

```diff
@@ -19,161 +19,42 @@
 
 {{/* If there are community pages... */}}
 {{- with $communities -}}
-  <section id="related_communities">
-    <header>
-      <h3>Join a Community</h3>
-    </header>
+  <article class="dg-promos dg-related-communities">
+    <h3 class="dg-promos__heading">Join a Community</h3>
 
-    <ul class="usa-list usa-list--unstyled">
+    <ul class="dg-promos__list">
       {{- range $communities -}}
-        <li class="promo">
-          <div class="promo__link-wrapper">
-            <a
-              href="{{ .Permalink }}?promo"
-              title="Visit {{ .Title }} community page"
-              >{{ .Title }}</a
-            >
-            {{- if .Params.members -}}
-              <p class="members">{{- .Params.members -}}+</p>
-            {{- end -}}
-          </div>
-        </li>
-      {{- end -}}
-    </ul>
-
-    <footer>
-      <p class="more">
-        <a href="{{ "/communities/" | absURL }}"
-          ><span
-            >See all
-            {{ len (where $.Site.RegularPages "Section" "communities") }}
-            communities</span
-          >
-          <svg
-            class="usa-icon dg-icon dg-icon--standard margin-bottom-05"
-            aria-hidden="true"
-            focusable="false"
+        <li class="promo dg-promos__list-item">
+          <a
+            href="{{ .Permalink }}?promo"
+            title="Visit {{ .Title }} community page"
+            class="dg-promos__link"
```

Comment:
Removed the count for consistency

<img width="270" alt="image" src="https://github.com/GSA/digitalgov.gov/assets/3385219/b5a165a6-6621-4057-9eae-060a1aa1833a">